src/browser.ts

```diff
@@ -1,14 +1,18 @@
 import {BaseIdrissCrypto} from "./baseIdrissCrypto";
 
 export class IdrissCrypto extends BaseIdrissCrypto {
-    constructor(polygonEndpoint: string = "https://polygon-rpc.com/", connectionOptions: ConnectionOptions = {}) {
+    constructor(twitterApiKey: string, polygonEndpoint: string = "https://polygon-rpc.com/", connectionOptions: ConnectionOptions = {}) {
```

Comment:
this is breaking change, lets make it a last optional argument

even better - put this in connection options as I mentioned above